packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -849,7 +902,18 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
 
     // The width of the button and the menu are defined by the widest
     // item and the width of the hint.
-    final List<Widget> items = _enabled ? List<Widget>.from(widget.items) : <Widget>[];
+    List<Widget> items = <Widget>[];
+    if (_enabled && widget.selectedItemBuilder == null)
+      items = List<Widget>.from(widget.items);
+    else if (_enabled && widget.selectedItemBuilder != null)
+      items = widget.selectedItemBuilder(context).map((Widget item) {
+        return Container(
```

Comment:
I opted to wrap the items in a similar `Container` as `DropdownMenuItem` instead of using `DropdownMenuItem` itself since `DropdownMenuItem` takes a value that determines whether it is selected or not, and that may not make sense here